assets/javascripts/src/actions.es6

```diff
@@ -39,6 +43,32 @@ export function submitPayment(dispatch) {
         .catch(error => dispatch({ type: PAYMENT_ERROR, kind: 'network', error: error }));
 }
 
+export function paypalRedirect(dispatch) {
+    let state = store.getState();
```

Comment:
this isn't a blocker for merging, but it's better to use `const` here instead, only use `let` if you want to reassign the variable (`const` == `val`, `let` == `var`)